File: DSA Using Java/Exercise/DecimalToBinary.java
public class DecimalToBinary {
    public static void getBinary(int n){
        int binary = 0;
        int pow = 0;
        while(n > 0){
            int lastDigit = n%2;
            binary = binary + (int)(lastDigit*Math.pow(10, pow));
            n /= 2;
            pow++;
        }

        System.out.println("Binary = "+binary);
    }
    public static void main(String[] args) {
        int decimal = 5;
        getBinary(decimal);
    }
}

File: DSA Using C++/Algorithms/Recursion and Backtracking/decimal_to_binary.cpp
#include <iostream>
#include <limits.h>

using namespace std;

// Time: O(log n), Space: O(long n)
void decToBinRec(int d, string &res)
{
    if (d > 1)
    {
        decToBinRec(d / 2, res);
    }
    res += (d % 2) + '0';
}

string decimalToBinary(int decimal)
{
    string result = "";
    decToBinRec(decimal, result);
    return result;
}

int main()
{
    int d = INT_MAX; // 2147483647 = 1111111111111111111111111111111

    cout << "Binary(" << d << ") = " << decimalToBinary(d) << endl;

    return 0;
}